openlibrary/coverstore/warc.py

```diff
@@ -41,7 +41,7 @@ def consume_crlf():
             if header is None:
                 break
             yield LazyWARCRecord(self._file, self._file.tell(), header)
-            self._file.seek(int(header.data_length), 1)
+            self._file.seek(self._file.tell() + int(header.data_length), 0)
```

Comment:
This is because `seek()` is defined in Python 3 as absolute, but relative in Python 2? If that's right, and we're aiming for polyglot code, then perhaps we need to have an `if PY3` or something.

Reply:
The commit message talks about the OSError that gets raise.  It is possible to create two separate codepaths but usually I first seek a common codepath to keep the A/B testing as clean as possible.